packages/next/server.js

```diff
@@ -3,4 +3,10 @@ module.exports = {
     .NextRequest,
   NextResponse: require('next/dist/server/web/spec-extension/response')
     .NextResponse,
+  parseUserAgentFromString:
```

Comment:
one of the goal of this PR is to reduce the bundle size since the library will be only included if you require it, but for that this file should be ESM